suite_held_suarez_1994.xml

```diff
@@ -3,6 +3,8 @@
 <suite name="held_suarez" version="1.0">
   <group name="physics">
     <scheme>held_suarez_1994</scheme>
-    <scheme>held_suarez_update</scheme>
+    <scheme>apply_tendency_of_x_wind</scheme>
+    <scheme>apply_tendency_of_y_wind</scheme>
+    <scheme>apply_heating_rate</scheme>
```

Comment:
Probably not an issue, but I thought I should point out. The held_suarez_update does not update the total tendencies, which these schemes also do.

Reply:
`physics_update` _does_ update total tendencies so I think it is correct for these routines to do so.

Reply:
Agreed - it is just the Held_Suarez didn't need it